File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cc
#include <stdio.h>
#pragma warning(disable:4996) 
int main() {
	int n = 0, subject = 0, sum = 0, count = 0;
	double avg = 0;
	int score[1001];

	scanf("%d", &n);

	for (int i = 0; i < n; i++) {
		
		int sum = 0, count = 0;
		double avg = 0;
		scanf("%d", &subject);

		for (int j = 0; j < subject; j++) {

			scanf("%d", &score[j]);
			sum += score[j];
		}

		avg = ((double)sum / subject);

		for (int k = 0; k < subject; k++) {

			if (score[k] > avg) {
				count++;
			}
		}

		printf("%.3lf%%\n", ((double)count / subject) * 100);
	}
}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
c = int(input())

for _ in range(c):
    subject = list(map(int, input().split()))
    avg = sum(subject[1:]) / subject[0]
    check = 0
    for j in subject[1:]:
        if(j > avg):
            check += 1

    result = (check / subject[0]) * 100
    print("%0.3f"%result+"%")